### ISSUE TITLE
Non-aligned training and intelligible audio

### ISSUE THREAD

**Issue by fabianoluzbr** [NVIDIA/tacotron2#390]
Dears,

I am trying to use Tacotron2 NVIDIA to train a model for Portuguese, however I cannot get alignment in the training. I use the same original settings, only the dataset that is in Brazilian Portuguese. The original dataset was 44.1KHz, I downsampled to 22050 Hz. This dataset has more than 10 hours of transcribed audio.

The Validation Loss on step 30000 is 0.61, attached some images of the training at some points. I appreciate any help.

![graph_3000](https://user-images.githubusercontent.com/29659419/88387576-26007800-cd89-11ea-95ac-7bb4e7118499.png)
![graph_15000](https://user-images.githubusercontent.com/29659419/88387582-2862d200-cd89-11ea-964a-fed47a112345.png)
![graph_5000](https://user-images.githubusercontent.com/29659419/88387583-28fb6880-cd89-11ea-8060-1dc403ae099b.png)
![graph_30000](https://user-images.githubusercontent.com/29659419/88387584-2993ff00-cd89-11ea-8ffb-6d4e70fdc4bd.png)

Here, my hparams:

`
import tensorflow as tf
from text import symbols

def create_hparams(hparams_string=None, verbose=False):
    """Create model hyperparameters. Parse nondefault from given string."""

    hparams = tf.contrib.training.HParams(
        ################################
        # Experiment Parameters        #
        ################################
        epochs=500,
        iters_per_checkpoint=1000,
        seed=1234,
        dynamic_loss_scaling=True,
        fp16_run=False,
        distributed_run=False,
        dist_backend="nccl",
        dist_url="tcp://localhost:54321",
        cudnn_enabled=True,
        cudnn_benchmark=False,
        ignore_layers=['embedding.weight'],

        ################################
        # Data Parameters             #
        ################################
        load_mel_from_disk=False,
        training_files='filelists2/metadata_train.txt',
        validation_files='filelists2/metadata_val.txt',
        text_cleaners=['portuguese_cleaners'],

        ################################
        # Audio Parameters             #
        ################################
        max_wav_value=32768.0,
        sampling_rate=22050,
        filter_length=1024,
        hop_length=256,
        win_length=1024,
        n_mel_channels=80,
        mel_fmin=0.0,
        mel_fmax=8000.0,

        ################################
        # Model Parameters             #
        ################################
        n_symbols=len(symbols),
        symbols_embedding_dim=512,

        # Encoder parameters
        encoder_kernel_size=5,
        encoder_n_convolutions=3,
        encoder_embedding_dim=512,

        # Decoder parameters
        n_frames_per_step=1,  # currently only 1 is supported
        decoder_rnn_dim=1024,
        prenet_dim=256,
        max_decoder_steps=1000,
        gate_threshold=0.5,
        p_attention_dropout=0.1,
        p_decoder_dropout=0.1,

        # Attention parameters
        attention_rnn_dim=1024,
        attention_dim=128,

        # Location Layer parameters
        attention_location_n_filters=32,
        attention_location_kernel_size=31,

        # Mel-post processing network parameters
        postnet_embedding_dim=512,
        postnet_kernel_size=5,
        postnet_n_convolutions=5,

        ################################
        # Optimization Hyperparameters #
        ################################
        use_saved_learning_rate=False,
        learning_rate=1e-3,
        weight_decay=1e-6,
        grad_clip_thresh=1.0,
        batch_size=4,
        mask_padding=True  # set model's padded outputs to padded values
    )

    if hparams_string:
        tf.logging.info('Parsing command line hparams: %s', hparams_string)
        hparams.parse(hparams_string)

    if verbose:
        tf.logging.info('Final parsed hparams: %s', hparams.values())

    return hparams

`

**Comment by Cris140** [comment]
You are training from scratch using Portuguese Corpus?
